src/generate.rs

```diff
@@ -79,7 +79,7 @@ pub fn device(
     });
 
     if let Some(cpu) = d.cpu.as_ref() {
-        let bits = util::unsuffixed(cpu.nvic_priority_bits as u64);
+        let bits = util::hex(cpu.nvic_priority_bits);
```

Comment:
I don't think writing this one in hex is going to make it more readable as the value is always in the range from 1 to 8.

Reply:
I prefer to have everything base 2/16 (as implied by the word "bits" here) related written in hex but I do agree it makes little sense for small numbers. I can get rid of that.